From 4c9e149d641a9945c3a8e15707b8712834f08c4f Mon Sep 17 00:00:00 2001
From: "Yann E. MORIN" <yann.morin.1998@free.fr>
Date: Sun, 22 Jan 2017 14:46:28 +0100
Subject: [PATCH] tests: add missing include

The int32_t type is defined in stdint.h.

The musl C library is very conservative in the headers that it
internally includes, and stdint.h is not included by any other header,
unlike with glibc or uClibc, which breaks the build.

Add the missing header.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>
---
Status: sent upstream
    https://lists.freedesktop.org/archives/wayland-devel/2017-January/032771.html
---
 tests/string-test.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/string-test.c b/tests/string-test.c
index a72ec30..5571b52 100644
--- a/tests/string-test.c
+++ b/tests/string-test.c
@@ -28,6 +28,7 @@
 #include <stdlib.h>
 #include <stdbool.h>
 #include <stdio.h>
+#include <stdint.h>
 #include <assert.h>
 
 #include "shared/string-helpers.h"
-- 
2.7.4

